public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -141,7 +141,7 @@ const getStyles = (theme: GrafanaTheme2, hasActions: boolean) => {
     content: css({
       display: 'flex',
       flexDirection: 'column',
-      paddingTop: hasActions ? TOP_BAR_LEVEL_HEIGHT * 2 : TOP_BAR_LEVEL_HEIGHT,
+      paddingTop: hasActions ? TOP_BAR_LEVEL_HEIGHT : 0,
```

Comment:
These changes and the ones in `AppChromeService` and `Page` handle getting rid of the extra space above the dashboard views that is normally reserved for the Grafana breadcrumb area and nav menu  button.